### PR TITLE
[prop-mvn-plugin] properties-maven-plugin

### PR DESCRIPTION
**Pull request by sk1418**

### CODE REVIEW

**Comment by theangrydev** [comment]
retest this please

**Comment by sk1418** [comment]
the verification maven plugin is configured in the parent pom... ..fixed

**Review by theangrydev — APPROVED** [review]

File: maven-modules/maven-plugins/external-properties-file/pom.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                </configuration>
+            </plugin>
+            <plugin>
+                <artifactId>maven-verifier-plugin</artifactId>
+                <version>${maven.verifier.version}</version>
+                <configuration>
+                    <verificationFile>../input-resources/verifications.xml</verificationFile>
+                    <failOnError>false</failOnError>
+                </configuration>
+            </plugin>
```
Comment (theangrydev) [inline maven-modules/maven-plugins/external-properties-file/pom.xml:37]: You can disable it instead like this:
```suggestion
            <plugin>
                <artifactId>maven-verifier-plugin</artifactId>
                <version>${maven.verifier.version}</version>
                <executions>
                    <execution>
                        <id>default</id>
                        <phase>none</phase>
                    </execution>
                </executions>
            </plugin>
```

File: maven-modules/maven-plugins/external-properties-file/pom.xml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                        <phase>initialize</phase>
+                        <goals>
+                            <goal>read-project-properties</goal>
+                        </goals>
+                        <configuration>
+                            <files>
+                                <file>props/user.properties</file>
+                                <!-- we can also pass in the file through mvn command line:
+                                <file>${user.properties.file}</file>
+                                -->
```
Comment (theangrydev) [inline maven-modules/maven-plugins/external-properties-file/pom.xml:54]: How about achieving both simultaneously by `<file>${user.properties.file}</file>` +  specify a default value for the property in the `<properties>` section?

**Review by theangrydev — CHANGES_REQUESTED** [review]

File: maven-modules/maven-plugins/external-properties-file/pom.xml (hunk 2 above)
Comment (sk1418) [reply maven-modules/maven-plugins/external-properties-file/pom.xml]: ✅

**Pull request merged**